pkg/driver/controller.go

```diff
@@ -179,8 +177,7 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, err
 	}
 
-	gidStr := strconv.Itoa(gid)
-	rootDirName := RootDirPrefix + "-" + gidStr + "-" + uuid.New().String()
+	rootDirName := volName
```

Comment:
this is the PVC name, not PV name I think. PVC name is not unique, at a given time u can have 2 PVC with same name in different namespace. should a unique string still be appended to be safe?

Reply:
nvm, i double checked it is PV name.

Reply:
since we use it as the clienttoken and handle AlreadyExists correctly, this LGTM.